### ISSUE TITLE
Chrome Responsive Design mode breaks `useDrag`

### ISSUE THREAD

**Issue by fightwithcrayon** [react-spring/react-use-gesture#163]
When Chrome Dev Tools' Responsive Design mode has been activated, `useDrag` no longer responds to interactions/fires at all.

This may be a bug in Chrome Dev Tools, but only ever seen this behaviour with this package.

To reproduce:

1. Open Chrome Dev Tools, disable Responsive Design Mode so that it no longer automatically opens next time we open Chrome Dev Tools. Close Chrome Dev tools, refresh the current page to be sure.
2. Open a page with `useDrag` (example link below)
3. Try dragging object, will work fine
4. Open Chrome Dev Tools. Should still work
5. Open Responsive Design Mode. 
6. Try dragging object. Will no longer work.
7. Close Responsive Design mode.
8. Try dragging object. Will still not work. Will require the page to be reloaded before it works again.

Chrome version: 81.0.4044.113
Reproducible when running this example: https://codesandbox.io/embed/react-three-fiber-gestures-dh2jc

A few more details:

- Have tried setting `pointerEvents` to true/false
- If Chrome Dev tools is open (without responsive design mode) when the page loads, works fine
- BUT, once responsive design mode has been enabled, `useDrag` does not start working again even if that feature is disabled/closed. Full reload is required.
- Behaviour happens regardless of whether selected Device Profile is set to Desktop/Desktop w. Touch/Mobile/Mobile w.o touch

**Comment by dbismut** [comment]
I'll have to look in details as this is a pretty old example using old versions of react-three-fiber and react-spring.

NB: In the meantime, I've released 7.0.15 as I fixed a very minor bug that has no incidence on this particular issue.

**Comment by fightwithcrayon** [comment]
Thanks. I've looked and we're currently using `7.0.4` with react spring `8.0.27`.

**Comment by dbismut** [comment]
I really think this is related to react-three-fiber, as [pointer events work normally without it](https://codesandbox.io/s/react-use-gesture-simple-fbmif).

See:
https://github.com/react-spring/react-three-fiber/issues/323

And:
https://github.com/react-spring/react-three-fiber/issues/326

-- ccing @drcmda

**Comment by fightwithcrayon** [comment]
I should say that example was something I used to verify the behaviour I was seeing in my own project, which doesn't use r3f. I'll have to try and create a repro to demonstrate when I get chance.

**Comment by dbismut** [comment]
Please do as this is something that should be fixed. Make sure you’re using the latest version of react-use-gesture.

**Comment by dbismut** [comment]
Ok @fightwithcrayon @drcmda I just managed to fix this one.

**TLDR: add `touch-action: none` to the dragged element (or canvas in this case**.

It's unrelated to the lib, or rather is correlated to how pointer events are handled on mobile. When the body is scrolled, pointer events are automatically canceled. So in our case, the browser thought we wanted a scroll and therefore canceled pointer event listeners, resulting in our drag gesture to be ended.

`touch-action: none` makes sure the browser doesn't interpret touch actions on the canvas, so our pointer events are no longer canceled.

There would be ways to deal with this that wouldn't involve using touch-action, such as `event.preventDefault()` but that wouldn't work well with [React event pooling system](https://reactjs.org/docs/events.html#event-pooling).
